_posts/2017-10-05-denounce-segwit2x.md

```diff
@@ -0,0 +1,75 @@
+---
+# This file is licensed under the MIT License (MIT) available on
+# http://opensource.org/licenses/MIT.
+
+type: posts
+layout: post
+lang: en
+category: blog
+
+title: "Bitcoin.org to denounce Segwit2x"
```

Comment:
Prefer if it's called simply 2X, since it has nothing to do with Segwit. Calling it "Segwit2x" plays into their deception.